### PR TITLE
feat: add brazilian portuguese translation

### PR DESCRIPTION
**Pull request by matheusinit**
I implemented and added support to brazilian portuguese for the package. If there is something to improve, let me know it. Thanks and I hope be helping!

### CODE REVIEW

**Comment by lbenie** [comment]
@all-contributors please add @matheusinit for code

File: lib/i18n/supportedLanguages.ts [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
  */
-export const supportedLanguages = ['en', 'fr', 'es', 'cn', 'ja', 'de'] as const
+export const supportedLanguages = [
+  'en',
+  'fr',
+  'es',
+  'cn',
+  'ja',
+  'de',
+  'ptBr',
```
Comment (lbenie) [inline lib/i18n/supportedLanguages.ts:11]: I'm wondering if this should be `pt-br` instead.

Comment (matheusinit) [reply lib/i18n/supportedLanguages.ts:11]: @lbenie Thinking now, is a better approach and a more common one.

**Pull request merged**